torchvision/transforms/transforms.py

```diff
@@ -734,14 +733,14 @@ def __init__(self, size, scale=(0.08, 1.0), ratio=(3. / 4., 4. / 3.), interpolat
 
     @staticmethod
     def get_params(
-            img: Tensor, scale: Tuple[float, float], ratio: Tuple[float, float]
+            img: Tensor, scale: List[float], ratio: List[float]
```

Comment:
The changes in this function seem unrelated to the whole goal of the PR.

For the next time, can you please factor them out in a different PR?